evap/results/templates/results_evaluation_detail.html

```diff
@@ -32,68 +31,62 @@
-                        {% if not evaluation.can_publish_rating_results %}
-                            <button
-                                type="button"
-                                disabled
-                                class="btn btn-sm btn-light"
-                                data-bs-toggle="tooltip"
-                                data-bs-placement="bottom"
-                                title="{% blocktranslate %}The results of this evaluation have not been published because it didn't get enough votes.{% endblocktranslate %}"
-                            >
-                                {% if evaluation.course.is_private %}
-                                    {% translate 'Participant' %}
-                                {% else %}
-                                    {% translate 'Public' %}
-                                {% endif %}
-                            </button>
-                        {% else %}
-                            <a
-                                href="{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view=public"
-                                role="button"
-                                class="btn btn-sm btn-light{% if view == 'public' %} active{% endif %}"
-                                data-bs-toggle="tooltip"
-                                data-bs-placement="bottom"
-                                title="
-                                    {% if evaluation.course.is_private %}
-                                        {% translate 'Shows results available for the participants.' %}"
-                                        >
-                                        {% translate 'Participant' %}
-                                    {% else %}
-                                        {% translate 'Shows results available for everyone logged in.' %}"
-                                        >
-                                        {% translate 'Public' %}
-                                    {% endif %}
-                            </a>
-                        {% endif %}
+                <div class="row">
+                    <div class="col-auto ms-auto">
+                        <div class="btn-switch btn-switch-light my-auto d-print-none">
+                            <div class="btn-switch-label">
+                                {% translate 'General results' %}
+                            </div>
+                            <div class="btn-switch btn-group">
+                                <a
+                                    href="{% url 'results:evaluation_detail' evaluation.course.semester.id evaluation.id %}?view_general_results=full&view_contributor_results={{ view_contributor_results.value }}"
+                                    role="button"
+                                    class="btn btn-sm btn-light {% if not general_textanswers %} disabled {% elif view_general_results == ViewGeneralResults.FULL %} active {% endif %}"
```

Comment:
Typically we try to generate HTML output that doesn't contain superfluous spaces (which this currently would do). I'm not super obsessed about this, but if you touch the file again, maybe fix that (as you do below with `disabled active` and `active`